### ISSUE TITLE
Seat chooser - missing visual focus on selected seat

### ISSUE THREAD

**Issue by weboverhauls** [adamsilver/nostyle#21]
For the Seat Chooser component, when tabbing to a selected seat (green), it's missing a visual focus indicator (focus ring).

https://nostyle.onrender.com/components/seat-chooser

![image](https://user-images.githubusercontent.com/473400/234359972-5bf58336-8520-4006-8b51-7c69be6b21f8.png)

**Comment by adamsilver** [comment]
Oh nice spot, thank you!
